app/views/alchemy/admin/nodes/index.html.erb

```diff
@@ -43,20 +43,6 @@
 </div>
 
 <script>
-  $('.nodes_tree').on('click', '.node_folder', function() {
```

Comment:
This event delegation is now missing from the new implementation. See note above.